change-beta/@azure-communication-react-da2d5b43-20a8-4bc0-8cbe-09ceb376a06f.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```

Comment:
Make sure that there is the right number of changelogs, you should be able to fix this by running `rush changelog` again 😊